python/lsst/pipe/tasks/postprocess.py

```diff
@@ -38,14 +38,14 @@
 from .functors import CompositeFunctor, RAColumn, DecColumn, Column
 
 
-def flattenFilters(df, filterDict, noDupCols=['coord_ra', 'coord_dec'], camelCase=False):
+def flattenFilters(df, noDupCols=['coord_ra', 'coord_dec'], camelCase=False):
     """Flattens a dataframe with multilevel column index
     """
     newDf = pd.DataFrame()
-    for filt, filtShort in filterDict.items():
-        subdf = df[filt]
+    for band in set(df.columns.to_frame()['band']):
+        subdf = df[band]
```

Comment:
I'm not familiar with Dataframes or with `postprocess.py`. Is it guaranteed that `df.columns.to_frame()` contains `'band'`? Or does a lookup of a nonexistent column(?) return an empty collection?

Reply:
Yes, the dataframe that is built as part of these tasks is constructed with `'band'` as a column.